deploy/README.md

```diff
@@ -351,8 +351,49 @@ helm repo update \
    && helm install stable/fluent-bit --name fluent-bit --namespace sumologic -f fluent-bit-overrides.yaml
 ```
 
+## Step 4: Deploy Falco
+
+In this step, you will deploy Falco to detect anomalous activity and capture Kubernetes Audit Events.
+
+__NOTE__ Falco needs privileged container access to insert its kernel module to process events for system calls.
```

Comment:
Can we link to some Falco documentation that talks to this?

Reply:
What's the reason to add it to our collection process? Does our solution rely on it?

Reply:
> Can we link to some Falco documentation that talks to this?

There is no detailed documentation around the privileged container access. This is the most relevant page I have found, https://sysdig.com/blog/sysdig-falco/. In the section, "how Falco works" it talks a bit about it. I will link this one.

Reply:
> What's the reason to add it to our collection process? Does our solution rely on it?

Our collection doesn't depend on it. Falco provides additional information in addition to what we are doing, specifically behavioral activity to detect anomalous activity. Falco can detect and alert on any behavior that involves making Linux system calls. In addition, it can also retrieve k8s audit events.